packages/squiggle-lang/__tests__/library/list_test.ts

```diff
@@ -115,7 +136,7 @@ describe("List functions", () => {
       "myadd(acc,x)=acc+x; arr=[1,2,3]; List.reduce(arr, 0, myadd)",
       "6"
     );
-    testEvalToBe("List.reduce([1,2,3], 0, add)", "6", true);
```

Comment:
I realized that this ``true`` meant that all of the other list_tests weren't being run. After this, I found out that some were broken, so I fixed those.